device/accton/x86_64-accton_es9618xx-r0/sonic_platform/fan.py

```diff
@@ -195,23 +216,40 @@ def get_speed(self):
 
     def get_target_speed(self):
         """
-        Retrieves the target (expected) speed of the fan
+        Retrieves the target (requested) speed of the fan.
         Returns:
             An integer, the percentage of full fan speed, in the range 0 (off)
                  to 100 (full speed)
+        """
+        if self.is_psu_fan:
+            return 100
+        elif self.get_presence():
+            speed = 100
+            try:
+                speed = int(self._api_helper.read_txt_file(FAN_REQUESTED_PERCENTAGE))
+            except:
+                pass
+            return speed
+        else:
+            return None
 
-        Note:
-            speed_pc = pwm_target/255*100
-
-            0   : when PWM mode is use
-            pwm : when pwm mode is not use
+    def get_configured_speed(self):
```

Comment:
Renamed the API to _get_configured_speed(self) to indicate local/internal usage.